ddtrace/appsec/_constants.py

```diff
@@ -1,10 +1,16 @@
 import os
+import sys
+
+
+if sys.version_info >= (3, 8):
+    from typing import Literal  # noqa:F401
+else:
+    from typing_extensions import Literal  # noqa:F401
+
 from typing import Any
 from typing import Iterator
 
 from ddtrace.internal.constants import HTTP_REQUEST_BLOCKED
-from ddtrace.internal.constants import REQUEST_PATH_PARAMS
```

Comment:
why do we remove these imports?

Reply:
Ah to make them literals. Let's just keep these as they are, with the imports, and then when you change ddtrace.internal.constants, the globals here will be changed as well.